### PR TITLE
Add collection description in collection detail page

### LINKED ISSUE #640: Bug - Collection description not displayed on collection page
**Scenario executed**
Given a user is browsing NFT collections
When the user selects a collection
Then the collection page should load
And it should display the NFTs within the collection, the collection name and the Collection description

**Steps to reproduce the bug**

1. Navigate to create collection screen
2. Add a name and description
3. Create collection and sign transaction
4. Navigate to newly created collection

**Expected result**
Collection description that was entered should be displayed

#### Design
* If set, the collection description paragraph is placed at the position of the first NFT cards. 
* Text style is Body 1
* Min-width is 330 px, max-width is 448px

[Figma frame](https://www.figma.com/file/4P9h3fF0oDI5U72ghXovdk/Altair-NFT-Studio?node-id=738%3A5983)

###### Empty collection
* If the collection does not contain any NFT, a message is shown: `The collection does not contain any NFT`

[Figma frame](https://www.figma.com/file/4P9h3fF0oDI5U72ghXovdk/Altair-NFT-Studio?node-id=740%3A6284)

**Actual result**
No collection description is being displayed

- Alessio to change to new version

### PR DESCRIPTION
**Pull request by alessioscalici**
### Description

This pull request adds the collection description in collection detail page, as per latest discussions and figma spec

Closes #640 

### Approvals

- [ ] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### Screenshots

Screenshot 2022-03-03 at 11 58 38
Screenshot 2022-03-03 at 11 59 09
Screenshot 2022-03-03 at 11 59 25

### CODE REVIEW

File: nft-studio/src/pages/Collection.tsx [hunk 1]
```diff
@@ -71,11 +76,14 @@ const Collection: React.FC = () => {
           )}
         </>
       ) : (
-        <Shelf justifyContent="center" mt="15vh" textAlign="center">
-          <Text variant="heading2" color="textSecondary">
-            The collection does not contain any NFTs yet
-          </Text>
-        </Shelf>
+        <Stack ml={3} alignItems="flex-start">
```
Comment (armandomistral) [inline nft-studio/src/pages/Collection.tsx:79]: @alessioscalici, please remove the left-hand margin of 24px of this text. The design spec was wrong (I had forgotten removing it when moving the part down below the description). Thank you

**Review by armandomistral — CHANGES_REQUESTED** [review]
Thanks, @alessioscalici! When connecting "Bob" account, not Mint action is shown. If connected with Alice address, it's shown. Can you reproduce this? 
image

**Review by onnovisser — APPROVED** [review]

**Comment by alessioscalici** [comment]
> Thanks, @alessioscalici! When connecting "Bob" account, no Mint action is shown. If connected with Alice address, it's shown. Can you reproduce this? image

@armandomistral 
image

Only the collection owner can mint NFTs, and Alice is (was) the owner of that beautifully empty collection! So indeed it's a feature 😄

File: nft-studio/src/pages/Collection.tsx (hunk 1 above)
Comment (alessioscalici) [reply nft-studio/src/pages/Collection.tsx]: Sure, done!

**Pull request merged**